homeassistant/components/modbus/strings.json

```
@@ -68,5 +68,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_close_comm_config": {
+      "title": "\"close_comm_on_error:\" is deprecated",
+      "description": "Please remove parameter \"close_comm_on_error:\" from the modus entry in configuration.yaml."
```

```suggestion
      "title": "`close_comm_on_error` YAML configuration option is being removed",
      "description": "The key `close_comm_on_error` YAML configuration option is deprecated and will be removed.\n\nPlease remove `close_comm_on_error` from the modus entry in configuration.yaml and restart Home Assistant to fix this issue."
```

Please replace `modbus` and `close_comm_on_error` with translation placeholders

---

I added the place holder to the changes, but in the changed version I made due to a request from @joostlek.